### ISSUE TITLE
US International Keyboard Layout in Windows 10 fails at compose keypresses

### ISSUE THREAD

**Issue by ivanhernandez** [Kethku/neovide#712]
When you press quote key (or one of those keys that serve as composers of characters), instead of waiting for next key to be pressed before writing into buffer, it immediately writes the quote. The second key get correctly printed, but there's now an unexpected quote char before the one I wanted.

When I pressed `<'>` I did expect no ouput until next key press, but `'` was printed instead.
When I pressed `<'>+<a>` I did expect `á` being printed, but `'á` was printed instead.
When I pressed `<'>+<space>` I did expect `'` being printed, but `''` was printed instead.
When I pressed `<">+<a>` I did expect `ä` being printed, but `"ä` was printed instead. 
When I pressed `<^>+<a>` I did expect `â` being printed, but `^â` was printed instead. 

Possibly solved by #445.

**Comment by Kethku** [comment]
repeat of https://github.com/Kethku/neovide/issues/445. This should be fixed once the new keyboard api lands

**Comment by ivanhernandez** [comment]
I thought that would be the case, but wasn't sure, as I am using a US QWERTY keyboard (just happening to be using international layout). May I suggest rewording #445 title, so it is more evident that it is a bag for NON US QWERTY STANDARD LAYOUT INPUT related issues?

A great frontend for nvim. anyway. Thanks for the cool work (wish I knew enough Rust to help).

**Comment by Kethku** [comment]
Done. I highly recommend picking up rust :) I knew very little when I started this project either
